jobs/validate-kubeflow-microk8s/Jenkinsfile

```diff
@@ -60,15 +84,9 @@ pipeline {
         }
         always {
             setEndTime()
-            collectDebug(params.controller, juju_model)
-
         }
-        cleanup {
-            saveMeta()
```

Comment:
Is there a way we can still capture some of this information? I utilize this for reporting failures etc